.circleci/config.yml

```diff
@@ -72,6 +72,54 @@ jobs:
           name: Publishing docker image
           command: IMAGE_SHA=${CIRCLE_SHA1} IMAGE_TAG=${CIRCLE_TAG:-latest} make -j8 cloudbuild
 
+  deploy:
+    description: Patches target cluster configuration
+    executor:
+      name: go/default
+      tag: '1.14'
+    parameters:
+      cluster:
+        type: string
+      committer_name:
+        type: string
+        default: TriggerMesh Bot
+      committer_email:
+        type: string
+        default: bot@triggermesh.com
+    steps:
+      - attach_workspace:
+          at: ~/
+      - add_ssh_keys
+      - run: ssh-keyscan github.com >> ~/.ssh/known_hosts
+      - run:
+          name: Configuring git
+          command: |
+            git config --global user.name '<< parameters.committer_name >>'
+            git config --global user.email '<< parameters.committer_email >>'
+      - run:
+          name: Cloning config repository
+          command: git clone --single-branch git@github.com:triggermesh/config.git tmconfig
+      - run:
+          name: Updating overlays/<< parameters.cluster >>/sources manifests
+          working_directory: tmconfig/
+          command: |
+            sed -i overlays/<< parameters.cluster >>/sources/deployments.yaml \
+              -e "s|gcr.io/triggermesh/aws-event-sources-controller:.*|gcr.io/triggermesh/aws-event-sources-controller:"${CIRCLE_TAG:-${CIRCLE_SHA1}}"|g" \
+              -e "s|gcr.io/triggermesh/awscodecommitsource:.*|gcr.io/triggermesh/awscodecommitsource:"${CIRCLE_TAG:-${CIRCLE_SHA1}}"|g" \
+              -e "s|gcr.io/triggermesh/awscognitosource:.*|gcr.io/triggermesh/awscognitosource:"${CIRCLE_TAG:-${CIRCLE_SHA1}}"|g" \
+              -e "s|gcr.io/triggermesh/awsdynamodbsource:.*|gcr.io/triggermesh/awsdynamodbsource:"${CIRCLE_TAG:-${CIRCLE_SHA1}}"|g" \
+              -e "s|gcr.io/triggermesh/awskinesissource:.*|gcr.io/triggermesh/awskinesissource:"${CIRCLE_TAG:-${CIRCLE_SHA1}}"|g" \
+              -e "s|gcr.io/triggermesh/awssnssource:.*|gcr.io/triggermesh/awssnssource:"${CIRCLE_TAG:-${CIRCLE_SHA1}}"|g" \
+              -e "s|gcr.io/triggermesh/awssqssource:.*|gcr.io/triggermesh/awssqssource:"${CIRCLE_TAG:-${CIRCLE_SHA1}}"|g"
```

Comment:
could you implement a bash loop like this one https://github.com/triggermesh/knative-targets/blob/master/.circleci/config.yml#L106-L111 ?

Reply:
Will do

Reply:
@sameersbn [done](https://github.com/triggermesh/aws-event-sources/compare/f7e0054f329eae9b424ba25824fdc53b13e910d9..350c2fde1be9fb42426bb4c0e425c11ccb4288f7)